grc/templates/upload/_upload-files-section.html

```diff
@@ -120,6 +141,22 @@ <h2 class="govuk-heading-m">
             if (event.target.files.length > 0) {
                 document.getElementById('submit').innerText = `Upload ${event.target.files.length} ${event.target.files.length == 1 ? 'file' : 'files'}`
             }
+
+            /*try {
+                document.getElementById('submit').disabled = true
+                const file = event.target.files.item(0)
+                if (file.type == 'application/pdf') {
+                    const data = await file.text()
+                    isEncrypted = data.substring(data.lastIndexOf('<<'), data.lastIndexOf('>>')).includes('/Encrypt')   // data.includes('Encrypt')
+                    if (isEncrypted) {
+                        document.getElementById('enter_password').style.display = 'block'
+                    }
+                }
+            } catch {
+            }
+            finally {
+                document.getElementById('submit').disabled = false
+            }*/

```

Comment:
More commented code

Reply:
This bit (and the `<div id="enter_password">` above) don't work (because we don't have the associated python code to remove the password).
So I guess this isn't a section that you regularly test with, so can be removed?